smoke-test/src/test/java/org/opennms/smoketest/dcb/DcbMultipleNodesIT.java

```diff
@@ -0,0 +1,220 @@
+            requisitionInterface.setMonitoredServices(monitoredServiceList);
+
+            final List<RequisitionMetaData> metaDataList = new ArrayList<>();
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:config-type", DCB_CONFIG_TYPE));
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:username", DCB_USERNAME));
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:password", DCB_PASSWORD));
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:script-file", DCB_SCRIPT_NAME));
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:ssh-port", Integer.toString((SSH_PORT))));
+            metaDataList.add(new RequisitionMetaData("requisition", "dcb:schedule", schedule));
+
+            final RequisitionNode requisitionNode = new RequisitionNode();
+            requisitionNode.setForeignId("local_" + i);
+            requisitionNode.setNodeLabel("local_" + i);
+            requisitionNode.setLocation(null);
+            requisitionNode.setMetaData(metaDataList);
+            requisitionNode.putInterface(requisitionInterface);
+
+            requisition.insertNode(requisitionNode);
+        }
+
+        restClient.addOrReplaceRequisition(requisition);
+        restClient.importRequisition(FOREIGN_SOURCE);
+
+        final NodeDao nodeDao = STACK.postgres().dao(NodeDaoHibernate.class);
+        final IpInterfaceDao ipInterfaceDao = STACK.postgres().dao(IpInterfaceDaoHibernate.class);
+        final MonitoredServiceDao monitoredServiceDao = STACK.postgres().dao(MonitoredServiceDaoHibernate.class);
+
+        localNode = await()
+                .atMost(3, MINUTES)
+                .pollInterval(30, SECONDS)
+                .until(DaoUtils.findMatchingCallable(nodeDao, new CriteriaBuilder(OnmsNode.class).eq("foreignId", "local_1").toCriteria()), notNullValue());
+
+        localInterface = ipInterfaceDao.findPrimaryInterfaceByNodeId(localNode.getId());
+
+        localService = monitoredServiceDao.getPrimaryService(localNode.getId(), "DeviceConfig");
+    }
+
+    @Before
+    public void setup() throws Exception {
+        // Ensure there is no backup history for every test
+        final var deviceConfigDao = STACK.postgres().dao(DeviceConfigDaoImpl.class);
+        deviceConfigDao.deleteAll(deviceConfigDao.findAll());
+        deviceConfigDao.flush();
+    }
+
+    private static String getContainerInternalIpAddress(final GenericContainer<?> container) {
+        return Iterables.getOnlyElement(container.getContainerInfo()
+                        .getNetworkSettings()
+                        .getNetworks()
+                        .entrySet())
+                .getValue()
+                .getIpAddress();
+    }
+
+    @Test
+    public void testGetConfigOnSchedule() throws Exception {
+        KarafShell karafShell=new KarafShell(STACK.opennms().getSshAddress());
+        karafShell.runCommand("opennms:sync-node-cache");
```

Comment:
Is this really required? If yes, this sounds like a bug and deserves an according comment.

Reply:
Removed as was not helpful in this case.